gulpfile.js

```diff
@@ -415,7 +415,7 @@ var codegen = function(project, cb) {
                         ' -CodeGenerator ' + generator +
                         ' -Namespace ' + mappings[project].package +
                         ' -Input ' + specRoot + '/' + mappings[project].source +
-                        ' -outputDirectory ' + mappings[project].dir +
+                        ' -outputDirectory ' + mappings[project].dir + '/src/main/java/' + mappings[project].package.replace(/\./g, '/') +
```

Comment:
Looks like you are actually using the same value that is assigned to outputDir here (see line 401).